Composer/packages/client/src/components/Notifications/NotificationCard.tsx

```diff
@@ -112,13 +113,20 @@ const cardDescription = css`
   word-break: break-word;
 `;
 
-const linkButton = css`
-  color: #0078d4;
-  float: right;
-  font-size: 12px;
-  height: auto;
-  margin: 4px 0 4px 8px;
-`;
+const linkButton = {
+  root: {
+    padding: '0',
```

Comment:
Padding and border should be '0px'

Reply:
In CSS, `0` is equivalent to `0px` (or zero of any other unit).